### PR TITLE
Salve dies dierum gloria full LH version

### PR DESCRIPTION
**Pull request by Avika1804**

### CODE REVIEW

File: Conventus/inbox/Resp-Gaudete-iusti-in-Domino-CROCHU.gabc [hunk 1]
```diff
@@ -9,5 +9,5 @@ centering-scheme: english;
 %width: 4.5;
 %height: 11;
 %%
-(c4) Gau(d>)dé(CD!fvED)te(dOf) iu(ixD!H/ivGFhgh/iv)sti(h) (,) in(dh) Dó(hf/hhg)mi(gffgf)no,(f) (;) al(fg>)le(gffd)lú(d!ewfd~)ia(d) : +(:) Rec(gd/ghgjvIHig)tos(g) de(fgFEfwgfg)cet(gf) (`) col(fg>)lau(fe>)dá(d)ti(d!ewfED)o,(d!ewf) (;) al(d/fh~)le(hg/hf/gf/egf/gvECd!ewf/gffoe)lú(de!fvEDe)ia.(ed) 
-<sp>V/</sp>(::) Con(h)fi(h)té(h)mi(h)ni(hg/hggof) Dó(g)mi(g)no(hf) in(gh) cí(gh)tha(ixh/iioh)ra;(h) (;) in(hf~) psal(gh)té(ixhi)ri(h)o(h) de(h)cem(h) chor(h)dá(ixhi)rum(h) psál(hg/hg)li(hf)te(ixf!ghg/hih) e(hvGFEfwg/hv)i.(gf) +(::) Rec(gd/ghgjvIHig)tos(g) (::)
\ No newline at end of file
+(c4) Gau(d>)dé(CD!fvED)te(dOf) iu(ixD!H/ivGFhgh/iv)sti(h) (,) in(dh) Dó(hf/hhg)mi(gffgf)no,(f) (;) al(fg>)le(gffd)lú(d!ewfd~)ia(d) : +(:) Rec(gd/ghgjvIHig)tos(g) de(fgFEfwgfg)cet(gf) (`) col(fg>)lau(fe>)dá(d)ti(d!ewfED)o,(d!ewf) (;) al(c/fh~)le(hg/hf/gf/egf/gvECd!ewf/gffoe)lú(de!fvEDe)ia.(ed)
```
Comment (jakubjelinek) [inline Conventus/inbox/Resp-Gaudete-iusti-in-Domino-CROCHU.gabc:12]: This is not going to apply, the files in inbox are gone each time I incorporate them.
Could you please revert it?  I'll fix it in my copy.

Comment (Avika1804) [reply Conventus/inbox/Resp-Gaudete-iusti-in-Domino-CROCHU.gabc:12]: yes, but how?

Comment (jakubjelinek) [reply Conventus/inbox/Resp-Gaudete-iusti-in-Domino-CROCHU.gabc:12]: For this time I've dealt with it, but one needs readd the removed inbox files, push, merge pull request and then remove that file again.
git revert commithash
should revert a commit, git log shows the hashes.
Or even better, before you start adding new things to git, do git pull to grab the changes and you'd see that the file is gone.

**Comment by Avika1804** [comment]
It sounds like advanced Chinese for me.

What should I do (or not do) to avoid this problem in the future

**Pull request merged**